README.md

```diff
@@ -71,35 +71,63 @@ else:
 
 ## Installation
 
-Create and activate a virtual environment and then install `asfquart` using [pip](https://pip.pypa.io):
+Create and activate a virtual environment and then install `asfquart` using [uv](https://docs.astral.sh/uv/) or [pip](https://pip.pypa.io):
 
 ```shell
-pip install "asfquart"
+# With uv
+uv add asfquart
+
+# With uv pip-compatible interface
+uv pip install asfquart
+
+# With standard pip
+pip install asfquart
 ```
 
 Note: Adding the `[aioldap]` extra will install optional dependencies for LDAP support that will
 require additional [system dependencies](https://github.com/noirello/bonsai?tab=readme-ov-file#requirements-for-building):
 
 ```shell
+# With uv
+uv add asfquart --extra aioldap
+
+# With uv pip-compatible interface
+uv pip install "asfquart[aioldap]"
+
+# With standard pip
 pip install "asfquart[aioldap]"
 ```
 
-## Building asfquart package
+## Development
 
-Prerequisites:
+### Install development environment
 
-- `poetry`: install e.g. with pipx `pipx install poetry`
+Install the required dependencies for development:
 
-Building the package:
+```shell
+uv sync
+```
+
+Install the optional dependencies for development:
 
 ```shell
-poetry build
+uv sync --extra aioldap
 ```
 
+### Building asfquart package
+
 Running the tests:
 
 ```shell
-poetry run pytest
+uv sync --group test
```

Comment:
```suggestion
uv sync --extra aioldap --group test
```

if the test also cover the extra, this would be needed. If not, we should be good.

Reply:
Done.
I guess the test should cover all extras including the further extras. Furthermore, since I install all extras during the Action, I think it would be better to maintain consistency in the README.

```yml
      - name: Install dependencies
        run: uv sync --locked --all-extras --group test
```